src/Simplex/Chat/Library/Subscriber.hs

```diff
@@ -1353,18 +1372,15 @@ processAgentMessageConn vr user@User {userId} corrId agentConnId agentMessage =
         cancelRcvFileTransfer user ft >>= mapM_ (deleteAgentConnectionAsync user)
         throwChatError $ CEFileRcvChunk err
 
-    memberConnectedChatItem :: GroupInfo -> GroupMember -> CM ()
-    memberConnectedChatItem gInfo m =
+    memberConnectedChatItem :: GroupInfo -> GroupChatScopeInfo -> GroupMember -> CM ()
+    memberConnectedChatItem gInfo gcsi m =
       -- ts should be broker ts but we don't have it for CON
-      createInternalChatItem user (CDGroupRcv gInfo m) (CIRcvGroupEvent RGEMemberConnected) Nothing
-
-    groupDescriptionChatItem :: GroupInfo -> GroupMember -> Text -> CM ()
```

Comment:
why removed

Reply:
it had a single call site and only created indirection, it's moved to maybeCreateGroupDescrLocal instead with is reused